src/api/api.ts

```diff
@@ -152,3 +154,45 @@ function dataURItoFile(dataURI: string) {
   const blob = new Blob([buffer], { type: mimeString });
   return blob;
 }
+
+//Program Enrollment
+export function getPatientEnrolledPrograms(patientUuid: string) {
+  return openmrsFetch(
+    `${BASE_WS_API_URL}programenrollment?patient=${patientUuid}&v=custom:(uuid,display,program,dateEnrolled,dateCompleted,location:(uuid,display))`,
+  ).then(({ data }) => {
+    if (data) {
+      return data;
+    }
+    return null;
+  });
+}
+
+export function createProgramEnrollment(payload, abortController) {
+  if (!payload) {
+    return null;
+  }
+  const { program, patient, dateEnrolled, dateCompleted, location } = payload;
```

Comment:
Here you are also assuming that all the properties of the payload exist, can't you validate this against a type ? something like: `payload instanceof ProgramEnrolmentType` ?

Reply:
Done